tests/e2e/utils/killApp.ts

```diff
@@ -1,11 +1,13 @@
 import config from '../config';
 import execAsync from './execAsync';
 
-export default function (platform = 'android', packageName = config.APP_PACKAGE) {
+const killApp = function (platform = 'android', packageName = config.MAIN_APP_PACKAGE): Promise<void> {
```

Comment:
@ruben-rebelo Should we add type for param here
```
platform: string = 'android', packageName: string = config.APP_PACKAGE
```

cc @blazejkustra

Reply:
One more note here, in [this PR](https://github.com/Expensify/App/pull/30793) we remove APP_PACKAGE and add MAIN_APP_PACKAGE. But we miss to update here

Reply:
@DylanDylann I don't think so, if you add the type even lint complains about Type trivially inferred as we are directly assigning a string to the param.
Regarding the APP_PACKAGE part I've noticed that and I've changed it to MAIN_APP_PACKAGE.

Reply:
I see. But It doesn't relate to this PR so I give a note for more clear